src/components/Settings/Settings.test.tsx

```diff
@@ -1,38 +1,53 @@
 import React from "react";
-import { render, fireEvent } from "@testing-library/react-native";
+import { render, fireEvent, act } from "@testing-library/react-native";
 import { Settings } from "./Settings";
 import { mockNavigation, resetNavigation } from "../../test/helpers/navigation";
+import { useConfig } from "../../common/hooks/useConfig";
 
 jest.mock("../../common/navigation");
+jest.mock("../../common/hooks/useConfig");
+
+const mockUseConfig = useConfig as jest.Mock;
+const mockSetValue = jest.fn();
+mockUseConfig.mockReturnValue({
+  config: { network: "ropsten" },
+  setValue: mockSetValue
+});
 
 describe("Settings", () => {
   beforeEach(() => {
     resetNavigation();
   });
-  it("should render the header", () => {
+  it("should render the header", async () => {
     expect.assertions(2);
     const { queryByText, queryByTestId } = render(
       <Settings onResetDocumentData={() => {}} navigation={mockNavigation} />
     );
-    expect(queryByText("Settings")).not.toBeNull();
-    expect(queryByTestId("header-bar")).not.toBeNull();
+    await act(async () => {
+      await expect(queryByText("Settings")).not.toBeNull();
+      await expect(queryByTestId("header-bar")).not.toBeNull();
+    });
```

Comment:
I didn't try it out, but could we perhaps use `wait` rather than `act` since we aren't calling actions here

Reply:
Turns out don't need act or wait when you mock. Added those when it was not mocked and didn't remove them.